Tests fail with Python-3.10a7

A new issue has been detected on Fedora upstream working on python-3.10 testing for Fedora 35.

https://bugzilla.redhat.com/show_bug.cgi?id=1950271

```=================================== FAILURES ===================================
_________________ [doctest] pgactivity.handlers.duration_mode __________________
039 Return the updated duration mode matching input key.
040 
041     >>> from blessed.keyboard import Keystroke as k
042 
043     >>> duration_mode(k("42"), DurationMode.query)
Expected:
    <DurationMode.query: 1>
Got:
    DurationMode.query

/builddir/build/BUILD/pg_activity-2.1.4/pgactivity/handlers.py:43: DocTestFailure
___________________ [doctest] pgactivity.handlers.query_mode ___________________
070 Return the query mode matching input key or None.
071 
072     >>> import curses
073     >>> from blessed.keyboard import Keystroke as k
074 
075     >>> query_mode(k("42"))
076     >>> query_mode(k("1"))
Expected:
    <QueryMode.activities: 'running queries'>
Got:
    QueryMode.activities

/builddir/build/BUILD/pg_activity-2.1.4/pgactivity/handlers.py:76: DocTestFailure
__________________ [doctest] pgactivity.handlers.sort_key_for __________________
094     >>> from blessed.keyboard import Keystroke as k
095     >>> from pgactivity.types import QueryMode
096 
097     >>> flag = Flag.all()
098 
099     Unhandled key:
100     >>> sort_key_for(k("1"), QueryMode.activities, flag)
101 
102     In activities mode, 'm', 'w', 't', ... keys are handled:
103     >>> sort_key_for(k("m"), QueryMode.activities, flag)
Expected:
    <SortKey.mem: 2>
Got:
    SortKey.mem

/builddir/build/BUILD/pg_activity-2.1.4/pgactivity/handlers.py:103: DocTestFailure
__________________ [doctest] pgactivity.handlers.verbose_mode __________________
054 Return the updated query display mode (aka verbose mode) matching input
055     key.
056 
057     >>> from blessed.keyboard import Keystroke as k
058 
059     >>> verbose_mode(k("42"), QueryDisplayMode.truncate)
Expected:
    <QueryDisplayMode.truncate: 1>
Got:
    QueryDisplayMode.truncate

/builddir/build/BUILD/pg_activity-2.1.4/pgactivity/handlers.py:59: DocTestFailure
_____________________ [doctest] pgactivity.types.UI.column _____________________
543 Return the column matching 'key'.
544 
545         >>> ui = UI.make()
546         >>> ui.column("cpu")
Expected:
    Column(name='CPU%', template_h='%6s ', mandatory=False, sort_key=<SortKey.cpu: 1>)
Got:
    Column(name='CPU%', template_h='%6s ', mandatory=False, sort_key=SortKey.cpu)

/builddir/build/BUILD/pg_activity-2.1.4/pgactivity/types.py:546: DocTestFailure
_____________________ [doctest] pgactivity.types.enum_next _____________________
032 Return an increment value of given enum.
033 
034     >>> class Seasons(enum.IntEnum):
035     ...     winter = 1
036     ...     spring = 2
037     ...     summer = 3
038     ...     autumn = 4
039 
040     >>> enum_next(Seasons.winter)
Expected:
    <Seasons.spring: 2>
Got:
    Seasons.spring

/builddir/build/BUILD/pg_activity-2.1.4/pgactivity/types.py:40: DocTestFailure
=========================== short test summary info ============================
FAILED pgactivity/handlers.py::pgactivity.handlers.duration_mode
FAILED pgactivity/handlers.py::pgactivity.handlers.query_mode
FAILED pgactivity/handlers.py::pgactivity.handlers.sort_key_for
FAILED pgactivity/handlers.py::pgactivity.handlers.verbose_mode
FAILED pgactivity/types.py::pgactivity.types.UI.column
FAILED pgactivity/types.py::pgactivity.types.enum_next
================== 6 failed, 37 passed, 1 deselected in 0.33s ==================```

--- a/pgactivity/handlers.py
+++ b/pgactivity/handlers.py
@@ -40,10 +40,10 @@ def duration_mode(key: Keystroke, mode: DurationMode) -> DurationMode:
 
     >>> from blessed.keyboard import Keystroke as k
 
-    >>> duration_mode(k("42"), DurationMode.query)
-    <DurationMode.query: 1>
-    >>> duration_mode(k("T"), DurationMode.transaction)
-    <DurationMode.backend: 3>
+    >>> duration_mode(k("42"), DurationMode.query).name
+    'query'
+    >>> duration_mode(k("T"), DurationMode.transaction).name
+    'backend'
     """
     if key == keys.CHANGE_DURATION_MODE:
         return enum_next(mode)
@@ -56,10 +56,10 @@ def verbose_mode(key: Keystroke, mode: QueryDisplayMode) -> QueryDisplayMode:
 
     >>> from blessed.keyboard import Keystroke as k
 
-    >>> verbose_mode(k("42"), QueryDisplayMode.truncate)
-    <QueryDisplayMode.truncate: 1>
-    >>> verbose_mode(k("v"), QueryDisplayMode.wrap_noindent)
-    <QueryDisplayMode.wrap: 3>
+    >>> verbose_mode(k("42"), QueryDisplayMode.truncate).name
+    'truncate'
+    >>> verbose_mode(k("v"), QueryDisplayMode.wrap_noindent).name
+    'wrap'
     """
     if key == keys.CHANGE_DISPLAY_MODE:
         return enum_next(mode)
@@ -73,10 +73,10 @@ def query_mode(key: Keystroke) -> Optional[QueryMode]:
     >>> from blessed.keyboard import Keystroke as k
 
     >>> query_mode(k("42"))
-    >>> query_mode(k("1"))
-    <QueryMode.activities: 'running queries'>
-    >>> query_mode(k(code=curses.KEY_F3))
-    <QueryMode.blocking: 'blocking queries'>
+    >>> query_mode(k("1")).name
+    'activities'
+    >>> query_mode(k(code=curses.KEY_F3)).name
+    'blocking'
     """
     if key.is_sequence:
         try:
@@ -100,24 +100,24 @@ def sort_key_for(
     >>> sort_key_for(k("1"), QueryMode.activities, flag)
 
     In activities mode, 'm', 'w', 't', ... keys are handled:
-    >>> sort_key_for(k("m"), QueryMode.activities, flag)
-    <SortKey.mem: 2>
-    >>> sort_key_for(k("w"), QueryMode.activities, flag)
-    <SortKey.write: 4>
-    >>> sort_key_for(k("t"), QueryMode.activities, flag)
-    <SortKey.duration: 5>
-    >>> sort_key_for(k("c"), QueryMode.activities, flag)
-    <SortKey.cpu: 1>
+    >>> sort_key_for(k("m"), QueryMode.activities, flag).name
+    'mem'
+    >>> sort_key_for(k("w"), QueryMode.activities, flag).name
+    'write'
+    >>> sort_key_for(k("t"), QueryMode.activities, flag).name
+    'duration'
+    >>> sort_key_for(k("c"), QueryMode.activities, flag).name
+    'cpu'
 
     In other modes, the default sort key is always returned:
-    >>> sort_key_for(k("m"), QueryMode.waiting, flag)
-    <SortKey.duration: 5>
+    >>> sort_key_for(k("m"), QueryMode.waiting, flag).name
+    'duration'
 
     When flag does not match given sort key, return None:
     >>> flag ^= Flag.CPU
     >>> sort_key_for(k("c"), QueryMode.activities, flag)
-    >>> sort_key_for(k("m"), QueryMode.activities, flag)
-    <SortKey.mem: 2>
+    >>> sort_key_for(k("m"), QueryMode.activities, flag).name
+    'mem'
     >>> flag ^= Flag.MEM
     >>> sort_key_for(k("m"), QueryMode.activities, flag)
     """

--- a/pgactivity/types.py
+++ b/pgactivity/types.py
@@ -37,12 +37,12 @@ def enum_next(e: E) -> E:
     ...     summer = 3
     ...     autumn = 4
 
-    >>> enum_next(Seasons.winter)
-    <Seasons.spring: 2>
-    >>> enum_next(Seasons.spring)
-    <Seasons.summer: 3>
-    >>> enum_next(Seasons.autumn)
-    <Seasons.winter: 1>
+    >>> enum_next(Seasons.winter).name
+    'spring'
+    >>> enum_next(Seasons.spring).name
+    'summer'
+    >>> enum_next(Seasons.autumn).name
+    'winter'
     """
     return e.__class__((e.value % max(e.__class__)) + 1)
 
@@ -543,8 +543,8 @@ class UI:
         """Return the column matching 'key'.
 
         >>> ui = UI.make()
-        >>> ui.column("cpu")
-        Column(name='CPU%', template_h='%6s ', mandatory=False, sort_key=<SortKey.cpu: 1>)
+        >>> ui.column("cpu")  # doctest: +ELLIPSIS
+        Column(name='CPU%', template_h='%6s ', mandatory=False, sort_key=...)
         >>> ui.column("gloups")
         Traceback (most recent call last):
           ...

